### ISSUE TITLE
[YZHSDImageCache queryDiskCacheForKey:done:]_block_invoke

### ISSUE THREAD

**Issue by e90909900** [rs/SDWebImage#1932]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 8.3.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

  Decompressing images that are downloaded and cached can improve performance but can consume lot of memory.I will canche. Config. ShouldDecompressImages and downloder shouldDecompressImages set to no, but have no effect. Still images decompression method.
![image](https://user-images.githubusercontent.com/17735394/27169545-e22ca8c2-51dc-11e7-9cd1-738d84fcf1c2.png)

![image](https://user-images.githubusercontent.com/17735394/27169529-d05863b6-51dc-11e7-9151-67bf83e6b221.png)

**Comment by bpoplauschi** [comment]
You need to let us know which version of the library are you using. Why did you not fill these info in the issue template and just left them with default values?
More to that, probably your setting is not done properly, please debug and stop on that line, check the setting.
